src/NuGet.Status/NuGet.Status.csproj

```diff
@@ -196,6 +187,9 @@
       <HintPath>..\packages\Microsoft.Bcl.Async.1.0.168\lib\net40\Microsoft.Threading.Tasks.Extensions.Desktop.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Win32.Primitives, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Can we move to `PackageReference`? I'm doing in for CDN redirect 🐱

Reply:
I tried, but the automatic migration tool doesn't seem to support this project. Given that, and the complexity of other parts of this change, I would rather not do it as part of this change.